TouchPortalAPI/tools.py

```diff
@@ -47,7 +47,7 @@ def convertImage_to_base64(image, type="Auto", image_formats=["image/png", "imag
         if type == "Auto" or type == "Web":
             try:
                 r = requests.head(image)
-                if r.headers['content-type'] in image_formats:
+                if r.headers.get('content-type', 'image/png' if type == "Web" else '') in image_formats:
```

Comment:
I did this because not all url image contains `content-type` for url image. (Ik this is not related but.... It's a fix...)